### ISSUE TITLE
JoditEditor in document should show control bar

### ISSUE THREAD

**Issue by piotrzarzycki21** [Moonshine-IDE/Super.Human.Portal#5]
When we switch to Edit mode in document JoditEditor should show toolbar

**Comment by piotrzarzycki21** [comment]
I think I'm running into that issue I have raised -> https://github.com/xdan/jodit/issues/935

**Comment by JoelProminic** [comment]
We'll want to use the same enabled actions as in MyAccount (i.e. Correspondence > New Ticket).
![image](https://user-images.githubusercontent.com/24532731/215838235-14821b98-f848-439a-a6a3-29a9b057073c.png)

It looks like this was already done in the demo today.   We'll need to do some tests to make sure that the Java API can handle the conversion for all the features.

**Comment by JoelProminic** [comment]
We are still waiting on a response from the Jodit team.    I'm bumping this to the next release for now - the bar is visible for now.

**Comment by piotrzarzycki21** [comment]
I will review that one in the next milestone again. I have updated JoditEditor to newest one.

**Comment by JoelProminic** [comment]
We have moved to used DevExpress (#31) for the editor, so this is no longer relevant
